api/openapi.yaml

```diff
@@ -650,6 +650,17 @@ components:
             The configuration branch to use.  This will automatically set commit to master on the branch
             when the configuration is added.
           pattern: '^[^\s;]*$'
+        specialParameters:
+          type: object
+          description: |
+            Optional parameters that do not affect the configuration content or are only used in
+            special circumstances.
+          properties:
+            imsRequireDkms:
```

Comment:
Does this default to `false`?

Reply:
Not setting it is the same as not specifying the parameter when creating an IMS customization session.  Currently I believe that behavior is to default to false in IMS, but CFS does not have a default value.

Reply:
In that case, maybe the description should be more along the lines of "If specified, the <whatever> parameter will be specified to the IMS session with the specified value". That makes it clear that setting it here just determines what value is passed into IMS, and IMS can use it as it pleases. Maybe also mention that if this is not specified, the parameter is not specified for the IMS session.